src/pages/SonarFeed/SignalDetails.js

```diff
@@ -26,9 +26,12 @@ const SignalDetails = ({
   redirect,
   closeModal,
   id,
-  match = {}
+  match = {},
+  author = 'Santiment Team',
+  link = 'https://santiment.net',
+  likesCount = 0
 }) => {
-  const WrapperEl = isModal(match) ? 'div' : Panel
+  const WrapperEl = Panel
```

Comment:
If there is no condition, then there is no point for making `WrapperEl` alias for `Panel`. `Panel` could be used directly in the markup

Reply:
Will removed